extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1146,6 +1146,46 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       'image': true,
     },
   },
+
+  'ibeatanalytics': {
+    'requests': {
+      'host': 'https://ibeats.indiatimes.com',
+      'base': 'https://ibeats.indiatimes.com/iBeat/pageTrendlogAmp.html',
+      'pageview': '${base}?' +
+                '&h=ibeats.indiatimes.com' +
+                '&d=ibeats.indiatimes.com' +
+                '&url=${url}' +
```

Comment:
I do not know much about ibeatanalytics, but ${url} probably should use the default variable, like ${sourceUrl}. By using default variables, you do not have to set each variable separately when setting the tag.